airbyte-integrations/connector-templates/source-configuration-based/metadata.yaml.hbs

```diff
@@ -30,5 +30,6 @@ data:
   supportLevel: community
   documentationUrl: https://docs.airbyte.com/integrations/sources/{{dashCase name}}
   tags:
-    - language:low-code
+    - language:python
+    - lowcode:true
```

Comment:
I remember @ambirdsall was pushing the thing where we would have `language:low-code`. Anything changed on that?

Reply:
Following the grooming of https://github.com/airbytehq/airbyte-internal-issues/issues/2849 I will change this to `- cdk:low-code`